Dockerfile

```diff
@@ -1,16 +1,38 @@
-FROM ubuntu:22.04
-
-WORKDIR /metachain 
+FROM ubuntu:latest
 
+WORKDIR /metachain
+#
 # Requires copy the binary to `build` folder beforehand
-COPY build/* /metachain 
-
+COPY target/* /metachain
```

Comment:
Changing this file will also break the ci environment, I would be against changing these if we don't have to.

Reply:
No longer relavent, will resolve it.